src/adapter/breakpointPredictor.ts

```diff
@@ -61,8 +61,13 @@ export class BreakpointsPredictor {
     );
   }
 
-  @logPerf<BreakpointsPredictor>(m => ({ type: m.repo.constructor.name }))
   private async createInitialMapping(): Promise<MetadataMap> {
+    return logPerf(this.logger, `BreakpointsPredictor.createInitialMapping`, () =>
```

Comment:
Since logging is now dependency injected, we can't use a cute decorator to log perf metrics. Oh well.